repose-aggregator/components/filters/keystone-v2-filter/src/test/scala/org/openrepose/filters/keystonev2/KeystoneV2SchemaTest.scala

```diff
@@ -17,13 +17,12 @@
  * limitations under the License.
  * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
  */
-
-package org.openrepose.filters.keystonev2.config
+package org.openrepose.filters.keystonev2
```

Comment:
Was this just moved to keep it consistent with other filters? I can't really think of a reason that this would make a difference, so I assume it's just consistency and/or preference.